docs/human-interface-guidelines.md

```diff
@@ -657,153 +657,188 @@ A Dynamic Notebook is a way for an app to provide user-managable tabbing functio
 
-elementary Icons have a distinctive visual style that is inspired in part by the [Tango Icons](http://tango.freedesktop.org/Tango_Desktop_Project). Icons on elementary share several common design elements.
+<div style="width:100%;display:inline-block;text-align:center;">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/dialog-warning.svg">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/internet-chat.svg">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/multimedia-photo-manager.svg">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/multimedia-video-player.svg">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/preferences-desktop-online-accounts.svg">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/utilities-terminal.svg">
+</div>
 
-### Design Elements {#design-elements}
 
-#### Shape {#shape}
+### Metaphors
 
-Icons should have a distinctive shape/silhouette to improve recognition. The shape should not be too complicated, but not every icon should be a rounded rectangle.
+If you're creating a icon for a hardware device or a file type (such as those for MimeType or Device icons), its shape is typically a visual representation of its real-world counterparts. For example, the icon for a camera is a stylized camera.
 
-#### Outline {#outline}
+<div style="width:100%;display:inline-block;text-align:center;">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/camera-photo.svg">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/drive-harddisk.svg">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/input-mouse.svg">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/package.svg">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/text-html.svg">
+	<img class="hig-icon" src="/images/docs/human-interface-guidelines/icons/64/video-display.svg">
+</div>
 
-All elementary Icons, and shapes within the icons, are stroked with a thin outline to improve contrast. At low resolutions the stroke size is 1px. The stroke should _not_ scale along with the icon, it should remain 1px.
 
-The color of the outline is a dark variant of the key color of the icon. This is done by creating a stroke at 30% opacity of black such that the border of the background lines up at half pixels. This is what allows icons with a stroke to appear sharp on dark backgrounds as well as light ones.
+### Action Icons
 
-#### Highlights {#highlights}
+Action icons are used to represent common user actions, such as "delete", "play" or "save". These icons are mostly found in app toolbars, but can be found throughout the OS.
```

Comment:
Do we use oxford commas? We really should.

> ... such as "delete", "play"**,** or "save".

"mostly found" might be better phrased as "most often found", and you don't need to mention they are found elsewhere if you've already said "mostly", it implies there are others for you.